src/config.py

```diff
@@ -56,16 +56,13 @@ def is_autopatch_build_enabled_in_image_buildspec(buildspec_path=None):
     image_buildspec_object = Buildspec()
     image_buildspec_object.load(buildspec_path)
     autopatch_build_flag = image_buildspec_object.get("autopatch_build", "False").lower() == "true"
-    print(f"Here: {buildspec_path}")
     return autopatch_build_flag
 
 
 def is_autopatch_build_enabled(buildspec_path=None):
-    return (
-        parse_dlc_developer_configs("build", "autopatch_build")
-        or os.getenv("AUTOPATCH")
-        or is_autopatch_build_enabled_in_image_buildspec(buildspec_path=buildspec_path)
-    )
+    return parse_dlc_developer_configs(
```

Comment:
might want to remove this dev toml option as well, if is_autopatch_build_enabled_in_buildspec is already set to true. A bit confusing to have tow controllers for this behavior

Reply:
Yes, if possible, lets remove the autopatch_build flag from the dev toml and the condition over here.